weft/wefttest/wefttest_test.go

```diff
@@ -16,6 +16,34 @@ const (
 	maxAge86400 = "max-age=86400"
 )
 
+var defaultCsp = map[string]string{
```

Comment:
The `defaultCsp` and `strictCsp` here are misleading. 

From here, I thought we'll be testing against the default CSP settings for weft (which is applying to all projects using weft), but it's actually checking against 2 html header style strings convert to maps only. 

It would make sense if we can use `defaultCsp` in `handlers.go` here. If it's not possible due to different packages then give these two maps a more generalised name (eg. `checkHeaders` and `expectedHeaders`) might help.

Reply:
OK, changed